libs/datamodel/connectors/dml/src/default_value.rs

```diff
@@ -215,6 +225,22 @@ mod tests {
         assert!(!auto_increment_default.is_autoincrement());
     }
 
+    #[test]
+    fn default_value_is_uuid() {
+        let auto_increment_default = DefaultValue::Expression(ValueGenerator::new_uuid());
```

Comment:
the name auto_increment default in the two tests is confusing

Reply:
good catch! will change